### PR TITLE
health check & bug fix

### PR DESCRIPTION
**Pull request by ShanniLi**
For #9

r @Raynos @kriskowal @rf 
cc @xichen2020 @prashantv @jsu1212 @breerly @abhinav @blampe @junchaowu

### CODE REVIEW

File: meta.thrift [hunk 1]
```diff
@@ -0,0 +1,4 @@
+service Meta {
+  void health()
```
Comment (blampe) [inline meta.thrift]: boooooooool :P

File: onedirection.thrift [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+struct Location {
+    1: required double course
+    2: required double epoch
+    3: required double longitude
+    4: required double latitude
+}
+
+service OneDirection {
```
Comment (blampe) [inline onedirection.thrift]: Not meant to be checked in?

File: as-http.js [hunk 3]
```diff
@@ -37,7 +36,10 @@ function TCurlAsHttp(options) {
     self.remoteHostPort = options.remoteHostPort;
     self.serviceName = options.serviceName;
     self.logger = options.logger;
+    self.channel = options.channel;
```
Comment (Raynos) [inline as-http.js]: Can we refactor as-http in a seperate PR ?

File: meta.thrift (hunk 1 above)
Comment (Raynos) [inline meta.thrift]: 4 spaces.

File: package.json [hunk 4]
```diff
@@ -25,7 +25,7 @@
     "my-local-ip": "^1.0.0",
     "process": "^0.11.0",
     "readable-stream": "^1.0.33",
-    "tchannel": "^2.0.1"
+    "tchannel": "2.1.2"
```
Comment (Raynos) [inline package.json]: Let's keep this to `^`

File: meta.thrift [hunk 5]
```diff
@@ -0,0 +1,4 @@
+service Meta {
```
Comment (Raynos) [inline meta.thrift]: This file should really live in tchannel and be imported from there rather then copied into two places.

Let's move it there in the future.

Comment (ShanniLi) [inline meta.thrift]: will do

File: meta.thrift (hunk 1 above)
Comment (ShanniLi) [inline meta.thrift]: @blampe 4 vs 2 in the vote for void :)

Comment (Raynos) [inline meta.thrift]: Tcurl uses 4 spaces. You don't have a choice.

File: as-http.js (hunk 3 above)
Comment (ShanniLi) [inline as-http.js]: #13

File: meta.thrift (hunk 1 above)
Comment (blampe) [inline meta.thrift]: Our health checks currently return 'OK'.

They don't need to do that -- a 200 status code is sufficient, just like an un-errored frame is sufficient here. That "OK" is significant to humans, though (and this endpoint is hit by humans *a lot* when trying to debug issues). If I query a service's health and get `undefined` or `null` back, it's not immediately obvious that that represents a healthy state. `True` is immediately obvious.

**Comment by Raynos** [comment]
lgtm.

**Pull request merged**